src/operator/tensor/broadcast_reduce_op.h

```diff
@@ -1090,6 +1118,7 @@ inline void BroadcastComputeImpl(const nnvm::NodeAttrs& attrs,
   mxnet::TShape src_shape, dst_shape;
   BroadcastReduceShapeCompact(outputs[0].shape_, small, &dst_shape, &src_shape);
   Stream<xpu> *s = ctx.get_stream<xpu>();
+  bool enable_lt = dst_shape.Size() > INT_MAX;
```

Comment:
Let's remove this flag and check the performance gain. If indeed we need to handle int64 case differently we need a more holistic design.

Reply:
will do